test/new/db/cmd/feat_arithmetic

```diff
@@ -55,7 +55,7 @@ EOF
 CMDS=<<EOF
 ?v 0x10+0x30
 ?v 1+3*2
-?v (1+3)*2
+"?v (1+3)*2"
```

Comment:
again, parenthesis are special chars, so in the new parser they should be quoted. We could also think about giving them a special meaning, like the `$((...)` in bash. But for now, let's just quote everything.

Reply:
wtf why